app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -34,5 +34,11 @@ def t(key, interpolations = {})
     end
     helper HelperOverride
 
+    # Adds the 'My Works' breadcrumb; we only want this for logged-in users
+    # overrides https://github.com/samvera/sufia/blob/v7.3.1/app/controllers/concerns/sufia/works_controller_behavior.rb#L93
+    def add_breadcrumb_for_controller
+      super if current_ability.current_user.staff?
+    end
```

Comment:
I would either check `current_ability.current_user.persisted?` here, or create a logged_in? cover method that just does that (`persisted?`) -- **or** just leave it how it is now. Your call.